Mathlib/Algebra/Order/Floor.lean

```diff
@@ -543,8 +546,9 @@ theorem floor_div_nat (a : α) (n : ℕ) : ⌊a / n⌋₊ = ⌊a⌋₊ / n := by
   · exact cast_pos.2 hn
 #align nat.floor_div_nat Nat.floor_div_nat
 
+-- See note [no_index around OfNat.ofNat]
 theorem floor_div_ofNat (a : α) (n : ℕ) [n.AtLeastTwo] :
-    ⌊a / OfNat.ofNat n⌋₊ = ⌊a⌋₊ / OfNat.ofNat n :=
+    ⌊a / (no_index (OfNat.ofNat n))⌋₊ = ⌊a⌋₊ / OfNat.ofNat n :=
```

Comment:
Oh, actually this lemma isn't marked `@[simp]` (and same with a few above). Should it be? If not, it seems inconsistent to use `no_index`, because if we cared about `rw?` and `exact?`, we would've been using `no_index` on RHS's too.

Reply:
Ugh, I don't know. Cargo culting this porting note can only go so far, maybe we need a zulip thread.

Reply:
Since this isn't `@[simp]`, I'm gonna push a commit removing the `no_index` but adding a comment to remind us about `no_index` if we make them `@[simp]` in the future.

Reply:
Having thought about this a bit more, I think the rules apply even if its not `simp`, as you can always use `simp [not_a_simp_lemma]`

Reply:
Okay, I'll revert. It would be consistent then to require `no_index` on RHSs too, since you can always `simp [← not_a_simp_lemma]`, but let's eschew consistency for now to avoid churn since hopefully https://github.com/leanprover/lean4/issues/2867 will be addressed soon anyway.